app/controllers/jobs/photo_proofs_controller.rb

```
@@ -0,0 +1,81 @@
+class Jobs::PhotoProofsController < AdminController
+  before_action :authenticate_user!, except: [:new, :create]
+  before_action :get_user_session_from_kotisauth, except: [:new, :create]
+  before_action :authorize_employee!, except: [:new, :create]
+
+  before_action :set_job
+  before_action :redirect_if_missing_file, only: :create
+  before_action :set_photo_proof_details
+  before_action :set_photo_proof, except: [:create, :new, :index]
+
+  def index
+  end
+
+  def show
+  end
+
+  def new
+    set_photo_proof if params[:id] && flash.present?
+  end
+
+  def create
+    @photo_proof = PhotoProof.new(job: @job, photograph: params[:file].tempfile)
+    if @photo_proof.save && photo_proof_email_send
+      flash[:notice] = "Photo Proof sent:"
+      redirect_to new_job_photo_proof_path(job_id: @job.id, id: @photo_proof.id)
+    else
+      flash[:notice] = "There was an error uploading the proof. Please try again."
+      render :new
+    end
+  end
+
+  def resend
+    notice = "Photo proof resent."
+    notice = "Something went wrong." unless photo_proof_email_send
+    flash[:notice] = notice
+    redirect_to action: :show
+  end
+
+  def approve
+  end
+
+  def update
+    if @photo_proof.approvable?
+      params[:photo_proof][:approved] = true if params[:commit].downcase == "approve"
+      params[:photo_proof][:approved] = false if params[:commit].downcase == "reject"
+      params[:photo_proof][:reviewed_by_id] = @user_session.kotisauth_id
+      @photo_proof.update!(photo_proof_params)
```

Do we want to throw an exception here?  

We should probably just use `.update` instead of `.update!` since we're not rescuing from exceptions.